scripts/clear-all-faults.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# This shell script sets the group D-Bus objects in
+# /xyz/openbmc_project/State//Decorator/OperationalStatusManager
+# to true or false. Also Remove the critical association from
```

Comment:
Which is the code that is removing the association?

Reply:
I haven't implemented that yet in this script. I will remove it for now. thanks for pointing it